### ISSUE TITLE
[FEATURE] Cropping function

### ISSUE THREAD

**Issue by DebadityaPal** [hs2361/k-means-compressor#2]
We need support for cropping of images, implementation should be simple enough as it would involve simply slicing the array.

**Comment by XDRAGON2002** [comment]
Can you elaborate on the specifications of the crop ? To be precise the dimensions of the crop (start point,end point,width,height)

**Comment by hs2361** [comment]
Yes, we basically want you to create a function that will take in all of those parameters along with the image matrix, and then crop the image according to those dimensions. Add appropriate boundary checks wherever needed.

**Comment by XDRAGON2002** [comment]
Got it, in that case I would like to try and work on this issue

**Comment by hs2361** [comment]
Looks like we've already assigned #4 to you. I'd suggest you work on that first, and then we can assign this to you after that.

**Comment by XDRAGON2002** [comment]
Oh okay then

**Comment by hs2361** [comment]
Hi @XDRAGON2002, are you still working on this issue?

**Comment by XDRAGON2002** [comment]
@hs2361 Yes, I am. Due to some issues I have not been active as much, but would still like to work on this.
